.github/workflows/docker_build.yaml

```
@@ -0,0 +1,168 @@
+permissions:
+  contents: write # Needed for pushing tags
+  packages: write # Needed for pushing docker images to GHCR
+
+jobs:
+  tag_release:
+    runs-on: ubuntu-latest
+    outputs:
+      # Define output to pass the tag to the next job
+      new_tag: ${{ steps.tag_version.outputs.next_version }}
+    steps:
+      - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)